sunkit_image/coalignment/__init__.py

```diff
@@ -0,0 +1,4 @@
+from sunkit_image.coalignment.interface import coalign
+from sunkit_image.coalignment.match_template import match_template_coalign as _  # NOQA: F401
```

Comment:
```suggestion
from sunkit_image.coalignment.match_template import match_template_coalign as _  # NOQA: F401
```
Why even import a symbol here?  Why not simply `import sunkit_image.coalignment_match_template`?